app/precompiled.py

```diff
@@ -119,21 +125,21 @@ def validate_pdf_document():
     if not encoded_string:
         abort(400)
 
-    invalid_pages, message = get_invalid_pages_with_message(BytesIO(encoded_string))
+    message = get_invalid_pages_with_message(BytesIO(encoded_string))
     data = {
-        'result': len(invalid_pages) == 0,
+        'result': bool(not message)
```

Comment:
Does it make sense that result is True or False instead of Success or Fail?

Reply:
Success or Fail might be better, but I think ultimately it's not worth changing it now since we want to drop that endpoint soon anyway (it's the one only used by Platform Admins currently, and when we deploy single upload, it will be obsolete, right?)